metatron/src/main/java/uk/gov/ida/eidas/metatron/exceptions/MetatronClientException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.ida.eidas.metatron.exceptions;
+
+public class MetatronClientException extends RuntimeException {
```

Comment:
I find the naming of the exception a bit odd. If I saw it I'd think it came from the client we use to call Metatron from the other apps. Where we're throwing it, it's not really an issue with the client, more with the request that the client made.

Having said that we call the client `MetatronProxy` so maybe it's ok. Not sure why we call it `MetatronProxy` and not `MetatronClient` though....